src/pymorize/units.py

```diff
@@ -110,15 +98,24 @@ def handle_unit_conversion(da: xr.DataArray, rule: Rule) -> xr.DataArray:
     model_unit = rule.get("model_unit")
     from_unit = da.attrs.get("units")
     if model_unit is not None:
-        logger.debug(
+        logger.info(
             f"using user defined unit ({model_unit}) instead of ({from_unit}) from DataArray "
```

Comment:
```suggestion
            f"using user defined unit ({model_unit}) instead of ({from_unit}) from the original file"
```

Please, check that my suggestion is correct :)